gulpfile.js

```diff
@@ -87,16 +85,28 @@ function startNodemon(done) {
   });
 }
 
+function setAvailablePort(done) {
+  const defaultPort = parseInt(process.env.PORT) || config.port;
+
+  findAvailablePort(function(port) {
```

Comment:
Shall we return a promise here so Gulp can handle errors for us?

```suggestion
async function setAvailablePort() {
  const defaultPort = parseInt(process.env.PORT) || config.port;

  return findAvailablePort(function(port) {
```